data-model/src/main/scala/za/co/absa/enceladus/model/versionedModel/NamedVersion.scala

```diff
@@ -18,4 +18,4 @@ package za.co.absa.enceladus.model.versionedModel
 /**
  * V3 Wrapper for [[za.co.absa.enceladus.model.versionedModel.VersionedSummary]]
  */
-case class NamedVersion(name: String, version: Int, disabled: Boolean)
+case class NamedVersion(name: String, version: Int, disabled: Boolean = false)
```

Comment:
Why this here&now?

Reply:
This was added as a sensible default. Now, because NamedVersion has been used in pagination-tests more extensively, where this default coming handy was more obvious (repetitive usage).

Reply:
Fair enough, good reason!